### PR TITLE
ROX-14129,ROX-14130,ROX-14131,ROX-14132: Fix failing boolean policy test.

### PR DESCRIPTION
**Pull request by dhaus67**
## Description

The current test failed after three years when it was written, which we now hit :-).

## Checklist
- [x] Investigated and inspected CI test results
~~- [ ] Unit test and regression tests added~~
~~- [ ] Evaluated and added CHANGELOG entry if required~~
~~- [ ] Determined and documented upgrade steps~~
~~- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))~~

If any of these don't apply, please comment below.

## Testing Performed

- see CI.

### CODE REVIEW

**Comment by dhaus67** [comment]
Current dependencies on/for this PR:
* master
  * **PR #4253** <a href="https://app.graphite.dev/github/pr/stackrox/stackrox/4253" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/stackrox/stackrox/4253?utm_source=stack-comment).

File: pkg/booleanpolicy/evaluator/evaluator_test.go [hunk 1]
```diff
@@ -1344,7 +1344,7 @@ func TestDifferentBaseTypes(t *testing.T) {
 				FieldQueries: []*query.FieldQuery{
 					// This test will fail in three years, but if this test still matters then,
 					// we have other problems...
-					{Field: "BaseTS", Values: []string{">1000d"}},
+					{Field: "BaseTS", Values: []string{">2000d"}},
```
Comment (porridge) [inline pkg/booleanpolicy/evaluator/evaluator_test.go:1347]: Please change this as well as the comment to 1000y instead (or if that does not fit, 100y).

**Review by porridge — APPROVED** [review]
Thanks for tracking this down. One nitpick.

File: pkg/booleanpolicy/evaluator/evaluator_test.go (hunk 1 above)
Comment (dhaus67) [reply pkg/booleanpolicy/evaluator/evaluator_test.go:1347]: I oversaw this one before merging, created a follow-up for this: #4255.
Sorry!

**Pull request merged**